Respawn/DbAdapter.cs

```diff
@@ -151,6 +151,7 @@ public string BuildTableCommandText(Checkpoint checkpoint)
                     var args = string.Join(",", checkpoint.TablesToIgnore.Select(t => $"N'{t}'"));
 
                     commandText += " AND t.name NOT IN (" + args + ")";
+                    commandText += " AND (s.name + '.' + t.name) NOT IN (" + args.Replace("[","").Replace("]","") + ")";

```

Comment:
I don't think we should assume a kind of an identifier delimiter here.

Reply:
Well, it will replace it only if it exists :).
This is just if someone's flavor is to send `[schema].[table_name]` that brackets be removed.

Reply:
Hi guys, any update on getting this merged in?